app/controllers/references_controller.rb

```diff
@@ -46,16 +48,12 @@ def paper_params
     def set_paper_locator
       locator_type, locator_id, paper_title = paper_params.values_at(:locator_type, :locator_id, :title)
 
-      return redirect_to(:back, alert: 'No parameters entered') if locator_id.blank?
+      return redirect_to(:back, alert: "Identifier can't be blank.") if locator_id.blank?
 
       case locator_type
       when 'doi'
         @locator = DoiPaperLocator.new locator_id: locator_id
       when 'link'
-        messages = []
-        messages << 'You must enter a title.' if paper_title.blank?
```

Comment:
the reason I had this was because I wanted to make it clear if they added nothing for a link, they are required to enter both a URL and Title.  Though, if we disable the Submit button if nothing is entered, this shouldn't a problem so nevermind.